modules/delegates/components/DelegateOverviewCard.tsx

```diff
@@ -29,12 +25,55 @@ import { CoreUnitModal } from './modals/CoreUnitModal';
 import { CoreUnitButton } from './modals/CoreUnitButton';
 import DelegateTags from './DelegateTags';
 import Icon from 'modules/app/components/Icon';
+import { Icon as UIIcon } from '@makerdao/dai-ui-icons';
 import DelegateExpiryDate from 'modules/migration/components/DelegateExpiryDate';
+import { DialogOverlay, DialogContent } from 'modules/app/components/Dialog';
+import BoxWithClose from 'modules/app/components/BoxWithClose';
 
 type PropTypes = {
   delegate: Delegate;
 };
 
+const DelegateVotingStatsModal = () => {
+  const [overlayOpen, setOverlayOpen] = useState(false);
+
+  return (
+    <>
+      <Flex onClick={() => setOverlayOpen(true)} sx={{ cursor: 'pointer', ml: 2 }}>
+        <UIIcon name="info" color="primary" />
+      </Flex>
+      {overlayOpen && (
+        <DialogOverlay isOpen={overlayOpen} onDismiss={() => setOverlayOpen(false)}>
+          <DialogContent ariaLabel="Delegate voting stats info">
+            <BoxWithClose close={() => setOverlayOpen(false)}>
+              <Flex
+                sx={{
+                  flexDirection: 'column',
+                  justifyContent: 'center',
+                  alignItems: 'center'
+                }}
+              >
+                <Heading sx={{ mb: 3 }}>Delegate voting stats</Heading>
+                <Text>
+                  - Participation: The percentage of votes the delegate has participated in. It combines stats
+                  for polls and executives.
+                  <br />
+                  <br />- Communication: The percentage of votes for which the delegate has publicly
+                  communicated their reasoning in addition to voting. It combines stats for polls and
+                  executives.
+                  <br />
+                  <br />
+                  Both stats are updated weekly by the GovAlpha Core Unit.
+                </Text>
```

Comment:
This is a minor thing, but from a semantic HTML perspective, I think each of these sections (participation, communication, and the last line) should be in their own `<Text as="p>{text}</Text>` components. Right now the whole thing shows as one span, but a span is usually used for inline elements to mark up a part of a text. Not for entire paragraphs.

<img width="496" alt="Screenshot 2023-01-23 at 10 20 59 AM" src="https://user-images.githubusercontent.com/5225766/214004709-8698b6c4-36a1-4468-9591-57ccd832e6af.png">